### PR TITLE
[WIP] @orta => Credit Card Things

### LINKED ISSUE #422: Move to Stripe
Balanced is shutting down.

### PR DESCRIPTION
**Pull request by ashfurrow**
This deals with credit card things. Mainly the transition from Balanced to Stripe for payment processing. Also allows users to manually enter their expiry date year with two digits.

I'd like to add some tests, but would love feedback. 

Fixes #422.
Related to #303.
Related to #333.

### CODE REVIEW

File: Kiosk/Bid Fulfillment/RACFunctions.swift [hunk 1]
```diff
@@ -22,19 +16,22 @@ func centsToPresentableDollarsString(cents:AnyObject!) -> AnyObject! {
...
 }
 
-func isZeroLengthString(string:AnyObject!) -> AnyObject! {
+func isZeroLengthString(string: AnyObject!) -> AnyObject! {
     return countElements(string as String) == 0
 }
 
-func is4CharLengthString(string:AnyObject!) -> AnyObject! {
-    return countElements(string as String) == 4
+func isStringLengthIn(range: Range<Int>)(string: AnyObject!) -> AnyObject! {
```
Comment (orta) [inline Kiosk/Bid Fulfillment/RACFunctions.swift]: this is lovely

File: fastlane/Fastfile [hunk 2]
```diff
@@ -16,11 +16,8 @@ lane :oss do
     'MixpanelProductionAPIClientKey', 
     'MixpanelStagingAPIClientKey', 
     'CardflightAPIClientKey', 
-    'CardflightAPIStagingClientKey', 
     'CardflightMerchantAccountToken', 
-    'CardflightMerchantAccountStagingToken', 
-    'BalancedMarketplaceToken', 
-    'BalancedMarketplaceStagingToken']
+    'StripePublishableKey']
```
Comment (orta) [inline fastlane/Fastfile]: Is this updated in 1pass?

File: Kiosk/Bid Fulfillment/RACFunctions.swift (hunk 1 above)
Comment (ashfurrow) [inline Kiosk/Bid Fulfillment/RACFunctions.swift]: Thanks! Inspired by @kylef.

File: fastlane/Fastfile (hunk 2 above)
Comment (ashfurrow) [inline fastlane/Fastfile]: Yup yup!

**Comment by ashfurrow** [comment]
Ha, just noticed this was merged. I'm still writing tests!

File: Kiosk/Bid Fulfillment/RACFunctions.swift (hunk 1 above)
Comment (kylef) [inline Kiosk/Bid Fulfillment/RACFunctions.swift]: Nice, curry all the things! Saw your slides for Istanbul and they looked familiar :wink:.

**Pull request merged**